tasks/collect-data/collect-data.yaml

```diff
@@ -131,6 +140,25 @@ spec:
         DATA_PATH="$(params.subdirectory)/data.json"
         echo -n "$DATA_PATH" > "$(results.data.path)"
         echo "$merged_output" | tee "$(workspaces.data.path)/$DATA_PATH"
+    - name: collect-single-component-mode-data
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      env:
+        - name: "SNAPSHOT"
+          value: '$(params.snapshot)'
+      script: |
+        #!/usr/bin/env bash
+        set -ex
+      
+        # check if RP/RPA has a single-component mode
+        DATA_PATH="$(params.subdirectory)/data.json"
+        SINGLE_COMPONENT_MODE=$(jq -er '.singleComponentMode // "false"' "$(workspaces.data.path)/$DATA_PATH")
```

Comment:
Can't fail with a default, so `-e` redundant

Reply:
I had `-cer` before...imagine!

Reply:
fixed

Reply:
:slightly_smiling_face: I only know it because Martin has corrected me on it like 27 times

Reply:
`jq -e` will fail if the resulting value is `null` or `false`, but not `"false"` as a string :)
```sh
jq -er '.singleComponentMode // false' <<< '{}' ; echo $?
false
1
jq -er '.singleComponentMode // "false"' <<< '{}' ; echo $?
false
0
```